src/app.js

```diff
@@ -1,21 +1,23 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
 
-module.exports = sum;
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const normilizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const parts = normilizedURL.pathname.slice(1).split('/');
+  const query = Object.fromEntries(normilizedURL.searchParams.entries());
+
+  res.setHeader('content-type', 'application.json');
```

Comment:
Incorrect MIME type in the 'content-type' header. It should be 'application/json' instead of 'application.json'